src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -0,0 +1,99 @@
+# Secure Communications
+
```

Comment:
It would be good to have a description of the high level architecture of this feature so that it's easier for people to understand. Especially how the different components like KBS, CAA (acting as the ssh client),
ssh and sshproxy in the podvm are working together. 
The following pdf (with updates based on the implementation) can be added for easy reference - https://github.com/confidential-containers/cloud-api-adaptor/files/14896131/PP.Secure.Comms.pdf

Reply:
@bpradipt 

I added some more text and the pdf
Please have a look if this is now ok.